server/utils/viewUtils.ts

```diff
@@ -27,4 +27,43 @@ export default class ViewUtils {
       }),
     }
   }
+
+  static summaryListArgs(summaryListItems: SummaryListItem[]): SummaryListArgs {
+    return {
+      rows: summaryListItems.map(item => {
+        return {
+          key: {
+            text: item.key,
+          },
+          value: (() => {
+            if (item.isList) {
+              const html = `<ul class="govuk-list">${item.lines
+                .map(line => `<li>${ViewUtils.escape(line)}</li>`)
+                .join('\n')}</ul>`
+              return { html }
+            }
+            if (item.lines.length > 1) {
+              const html = item.lines.map(line => `<p class="govuk-body">${ViewUtils.escape(line)}</p>`).join('\n')
```

Comment:
Would be good to test that you get `<p>` tags when there are multiple lines and it's not a list.

Reply:
Have added this to the current test